test-data/local/externally_sampled_params/forward_model.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import os
+import json
+
+import numpy as np
+
+rng = np.random.default_rng()
+
+
+def _load_coeffs(filename):
+    with open(filename) as f:
+        return json.load(f)
+
+
+def _evaluate(coeffs, x):
+    return coeffs["a"] * x**2 + coeffs["b"] * x + coeffs["c"]
+
+
+if __name__ == "__main__":
+    a = rng.uniform(0, 1)
+    b = rng.uniform(0, 2)
+    c = rng.uniform(0, 5)
+
+    with open("externally_sampled_params0", "w") as f:
+        f.write(f"{a}\n{b}\n{c}")
+
+    if os.path.exists("params.json"):
```

Comment:
This might warrent a comment? Just remark that in the first iteration the sampling is done outside ert, while in later iterations we read the updated parameters into the forward model?

Reply:
Agree, thanks.